src/components/FavoriteCities/FavoriteCities.jsx

```diff
@@ -2,19 +2,43 @@
 import classes from './FavoriteCities.module.scss';
 import * as React from 'react';
 import FavoriteCityForecast from './FavoriteCityForecast/FavoriteCityForecast';
-import mockFavoriteCities from './mockData';
+import { connect } from 'react-redux';
+import EmptyListMessage from '../EmptyListMessage/EmptyListMessage';
+import type { FavoriteCitiesPropsType } from './FavoriteCitiesPropsType';
 
-function FavoriteCities(): React.Node {
+const makeFavoriteCitiesList = favoriteCities =>
```

Comment:
There is not much added value to create such a function as a separate in scope. You can use them inline